### DISCUSSION TITLE
InvertModel settings do not respond

### DISCUSSION THREAD

**Discussion by khashabri** [James-Yu/LaTeX-Workshop#3393] [Q&A]
I am trying to make the PDF view darker by using the `Invert mode` as described [here](https://github.com/James-Yu/LaTeX-Workshop/wiki/View#invert-mode). However, no matter what parameters I set, nothing changes in the PDF (see screenshot). Am I missing something?

Since I haven't found any other help, I had to post my question here. I hope someone can provide me with some help. 

![Bildschirmfoto 2022-07-23 um 22 29 16](https://user-images.githubusercontent.com/50481847/180622487-97f189d3-6fed-4ed7-8d09-40269e157349.png)

**Comment by khashabri** [comment]
Still no solution found. 
I consider this actually as a bug. Push it to questions if you want to make the issues section "clean".

**Comment by jlelong** [comment]
You have to reload the viewer to make any change in the invert mode settings effective.

**Reply by khashabri** [reply]
I have done this before. The only thing that changes are some pieces in the figures. Like for example in this image, the dots in the plots have all gone dark but not the whole PDF. Is this what the feature is supposed to do?
![Bildschirmfoto 2022-08-05 um 21 32 18](https://user-images.githubusercontent.com/50481847/183148910-811a0f11-8072-4461-83bc-11c08779761f.png)
![Bildschirmfoto 2022-08-05 um 21 34 29](https://user-images.githubusercontent.com/50481847/183148917-4aad48ca-c103-4069-ae62-b32901e04bea.png)
![Bildschirmfoto 2022-08-05 um 21 35 05](https://user-images.githubusercontent.com/50481847/183148922-9f2a139a-9bc5-45ab-a3b9-ab20e1f258a5.png)
